src/styles/graphql-tryout.css

```
@@ -135,6 +135,47 @@
+}
+
+.graphql-auth-tabs {
+  display: flex;
+  gap: 0;
+  border-bottom: 1px solid #d0d7de;
+  margin-bottom: 8px;
+}
+
+.graphql-auth-tab {
+  padding: 6px 16px;
+  font-family: 'Montserrat', sans-serif;
```

Stylelint (17.10.0)

[error] Expected no quotes around "Montserrat" (font-family-name-quotes)

(font-family-name-quotes)